pkg/testutils/release/releases.go

```diff
@@ -56,6 +56,32 @@ func parseReleases() (map[string]Series, error) {
 	return result, nil
 }
 
+// MajorReleasesBetween returns the number of major releases betweeen
+// the two versions passed. Returns an error when there is no
+// predecessor information for a release in the chain (which should
+// only happen if one of the versions passed is very old).
+func MajorReleasesBetween(v1, v2 *version.Version) (int, error) {
```

Comment:
Nit: `v1` -> `from`, `v2` -> `to`.

Reply:
Nm, since the order isn't required, maybe update the comment to say `...between _any_ two versions passed`.

Reply:
Done.